style.css

```diff
@@ -803,3 +729,601 @@ button {
   border-radius: 10px;
   margin-left: 120px;
 }
+
+@media (max-width: 768px) {
+  .deskmenu {
+    display: none;
+  }
+
+  .cardwork1 .portfolio1 {
+    display: none;
+  }
+
+  .logo {
+    display: flex;
+    flex-direction: row;
+  }
+
+  .logo a {
+    width: 76px;
+    height: 20px;
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 15px;
+    line-height: 20px;
+    text-decoration: none;
+    display: flex;
+    align-items: center;
+    text-align: center;
+    color: #6070ff;
+  }
+
+  .toggle {
+    display: flex;
+    align-items: flex-end;
+    padding-left: 70%;
+  }
+
+  .headerline {
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start;
+    padding: 114px 24px;
+    gap: 12px;
+    width: 100%;
+    height: 604px;
+    left: calc(50% - 375px / 2);
+    top: 92px;
+    position: absolute;
+  }
+
+  .headerinside {
+    width: 100%;
+    background-image: url(images/header-shapes.png);
+    background-repeat: no-repeat;
+    background-size: cover;
+  }
+
+  .portfoliocards {
+    display: flex;
+    flex-direction: column;
+  }
+
+  .cardwork1 {
+    display: flex;
+    flex-direction: column;
+  }
+
+  .cardwork2 {
+    display: flex;
+    flex-direction: column;
+  }
+
+  .cardwork3 {
+    display: flex;
+    flex-direction: column;
+  }
+
+  .form-button {
+    margin-left: 20px;
+    background-repeat: no-repeat;
+    background-position: center, right;
+  }
+
+  .see {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    padding: 12px 16px;
+    position: absolute;
+    width: 140px;
+    height: 48px;
+    background: #fff;
+    border-radius: 8px;
+    border: transparent;
+  }
+
+  button {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    padding: 12px 16px;
+    position: absolute;
+    width: 140px;
+    height: 48px;
+    left: 24px;
+    border-radius: 8px;
+    margin-top: -20px;
+  }
+}
+
+@media (min-width: 768px) {
+  body {
+    width: 100%;
+    height: 5125px;
+    background: #f4f5f7;
+  }
+
+  .mobile {
+    position: absolute;
+    width: 100%;
+    height: 916px;
+    top: 10px;
+    background: #fff;
+  }
```

Comment:
- [ ] As required for this project milestone, your mobile version should also look as pretended, and if we check the mobile design, it appears to be a bit ugly; please improve this so that your project looks amazing and recruiters may contact you. it much more better if your mobile version match the figma design :wink: 

This is what your mobile version looks like point_down: 
![mobile version](https://user-images.githubusercontent.com/88894525/194707394-3eca9c4a-a0bc-4df6-98c1-cab2d38351cd.png)

Figma Design :point_down: 
![figmamobile](https://user-images.githubusercontent.com/88894525/194707473-391a1fc8-1724-4af3-b004-74f611fafec4.png)